examples/example.qbs

```diff
@@ -0,0 +1,27 @@
+import qbs
+import "../box2d.qbs" as Box2D
+
+Project {
+    name: "Example";
+
+    Box2D { }
+
+    Product {
+        files: ["main.cpp"];
+        name: "Example";
+        type: "application";
+
+        Depends { name: "cpp"; }
+        Depends { name: "Qt"; submodules: ["core", "qml", "quick"]; }
+        Depends { name: "box2d_lib"; }
+        Group   { name: "QML Files";    files: ["*.qml", "*/*.qml"]; }
+        Group   { name: "QRC Files";    files: ["*.qrc"] }
+        Group   { name: "Script Files"; files: ["*.js", "*/*.js"] }
+
+        cpp.includePaths: [".."]
```

Comment:
This line would not be necessary if you'd add the following to the "box2d_lib" product:
```qml
    Export {
        Depends { name: "cpp" }
        cpp.includePaths: "."
    }
```